templates/lightbox.html

```diff
@@ -0,0 +1,58 @@
+<style>
+    .lightbox {
+        position: fixed;
+        z-index: 1000;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        background-color: rgba(22,27,34,0.8);
+        display: none;
+    }
+
+    .lightbox-active {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+</style>
+<script>
+    let open_lightboxes = []
+
+    function createLightbox(id_lightbox, id_content){
```

Comment:
```suggestion
    function createLightbox(id_lightbox, id_content) {
```
nit: space between the brace (`{`) and the end of the parameter list.